packages/kbn-i18n/src/react/provider.tsx

```diff
@@ -71,8 +75,19 @@ function wrapIntlFormatMessage(child: React.ReactNode) {
  */
 export class I18nProvider extends React.PureComponent {
   public static propTypes = { children: PropTypes.element.isRequired };
+  public WrapedIntlFormatMessage: any;
+
+  public constructor(props: any) {
```

Comment:
__issue:__ instead of `props: any` better introduce smth like this:
```ts
interface I18nProviderProps {
  children: React.ReactNode;
}

....
export class I18nProvider extends React.PureComponent<Readonly<I18nProviderProps>> {
.......
public constructor(props: Readonly<I18nProviderProps>) {
.......
```